arch/posix/eventloop_posix_tcp.c

```diff
@@ -8,6 +8,7 @@
 
 #include "open62541/types.h"
 #include "eventloop_posix.h"
+#include "../../src/server/ua_server_internal.h"
```

Comment:
Not a fan of this include but I could not find another way to get access to the SecureChannels list from this file. Is there a better way I'm not seeing?

Reply:
Yes, that include is a no-go. It breaks abstraction boundaries.
Can't you trigger the purging from within the server?

Reply:
Yes, I managed to move everything to within the server